src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+    private List<Ball> balls = new ArrayList<>();

```

Comment:
We do not need this functionality, it's out of task scope.
I suggest you to watch this video https://www.youtube.com/watch?v=Ot2eB07rjcI&t=508s